recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ build:
   number: {{ build }}
   skip: true  # [win]
   skip: true  # [linux and cuda_compiler_version != "11.8"]
+  rpaths_patcher: patchelf  # [linux]
```

Comment:
This should be unneeded, let's remove it once we confirm rebuilding rdma-core was the last blocker
```suggestion
```